test/test_python_dispatch.py

```diff
@@ -85,7 +85,7 @@ def emit(self, record):
         self.log_list.append(f'{fmt_rets} = {record.msg}({fmt_args})')
 
 def log_input(name: str, var: object):
-    logging.getLogger("LoggingTensor").info("input", (name,), {}, (var,))
+    logging.getLogger("LoggingTensor").info("input %s %s %s", (name,), {}, (var,))
```

Comment:
This leads to the following failure on my machine:
```
======================================================================
FAIL: test_basic (__main__.TestPythonDispatch)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test/test_python_dispatch.py", line 128, in test_basic
    $5 = torch._ops.aten.add($4, $3)''')
  File "/raid/rzou/pt/workspace-env/lib/python3.7/site-packages/expecttest/__init__.py", line 262, in assertExpectedInline
    self.assertMultiLineEqualMaybeCppStack(expect, actual, msg=help_text)
  File "/raid/rzou/pt/workspace-env/lib/python3.7/site-packages/expecttest/__init__.py", line 281, in assertMultiLineEqualMa
ybeCppStack
    self.assertMultiLineEqual(expect, actual[:len(expect)], *args, **kwargs)
AssertionError: "$0 = input('x')\n$1 = torch._ops.aten.mul($0, $0)\n[118 chars] $3)" != "$0 = input %s %s %s('x')\n$1 = torc
h._ops.aten.mul([118 chars]._op"
- $0 = input('x')
+ $0 = input %s %s %s('x')
?           +++++++++
  $1 = torch._ops.aten.mul($0, $0)
- $2 = input('grad_y')
+ $2 = input %s %s %s('grad_y')
?           +++++++++
  $3 = torch._ops.aten.mul($2, $0)
  $4 = torch._ops.aten.mul($2, $0)
- $5 = torch._ops.aten.add($4, $3)+ $5 = torch._op : To accept the new output, re-run test with envvar EXPECTTEST_ACCEPT=1 (
we recommend staging/committing your changes before doing this)
```

It looks like test/test_python_dispatch is not properly set up to run in our CI

Reply:
This change is also just not right at all, rendering happens later